src/Api/Modules/PotStageHistory.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Zoho\CRM\Api\Modules;
+
+class PotStageHistory extends AbstractRecordsModule
+{
+    protected static $primary_key = 'POTENTIALSTAGEHISTORYID';
+
+    protected static $associated_entity = \Zoho\CRM\Entities\PotStageHistory::class;
+
+    protected static $supported_methods = [
+        'getRelatedRecords',
+    ];
+
+    public function getPotentialHistory($potentialId)
+    {
+        return $this->getRelatedById('Potential', $potentialId)->fetch()->getContent();
```

Comment:
The `->fetch()->getContent()` part must be called explicitely by the developer or is done automatically by the `Client`, therefore you must remove it.

To achieve the same thing automatically, you need to set these preferences:
1. `fetch()` is done by setting `'auto_fetch_paginated_requests'` to `true`.
2. `getContent()` is done by setting `'response_mode'` to `Zoho\CRM\ClientResponseMode::DIRECT`.

Notice the preference `'records_as_entities'`, used to transform records into entities automatically.